src/components/TableStateContainer.tsx

```diff
@@ -225,6 +226,7 @@ const TableStateContainerElement: React.FC<Props> = ({ gridTemplateColumns, colu
           marginTop: "16px",
         }}
       >
+        { textSearchBox !== false?
```

Comment:
This condition is a bit strange. Since textSearchBox is a boolean, you should be able say: `textSearchBox ? <TableSearchBox /> : null`. I understand that in case textSearchBox is undefined, you still want to show the search box but it's a bit unclear to do it this way. It should be the other way around, if you don't specify the textSearchBox parameter then the component will not show the search box at all.

Reply:
True, good point. improved the default here:
#598